app/controllers/api/v1/base_controller.rb

```diff
@@ -8,21 +8,35 @@ class BaseController < ApplicationController
       private
 
       def require_authentication!
-        user = User.find(request.headers["X-User-Id"])
+        user = find_user
 
         if user.present? && valid_token?(user.authentication_token)
           sign_in user
         else
-          render status: :unauthorized
+          render json: {}, status: :unauthorized
```

Comment:
Do you need to render anything here?
You're not testing for this.
Maybe use `header` instead of `render`

Reply:
uhh! `header` is perfect for this :3